packages/api/package.json

```diff
@@ -105,8 +108,14 @@
     "@axelarjs/core": "workspace:*",
     "@axelarjs/cosmos": "workspace:*",
     "@axelarjs/utils": "workspace:*",
+    "@axelarjs/proto": "workspace:*",
+    "clone-deep": "^4.0.1",
+    "cross-fetch": "^4.0.0",
+    "ethers": "^6.13.2",
     "isomorphic-unfetch": "^4.0.2",
     "rambda": "^9.1.1",
+    "string-similarity-js": "^2.1.4",
+    "standard-http-error": "^2.0.1",
```

Comment:
i think only `string-similarity-js` is necessary here. could you try removing others?